paramiko/server.py

```diff
@@ -301,9 +304,8 @@ def enable_auth_gssapi(self):
         :see: : `.ssh_gss`
         """
         UseGSSAPI = False
-        GSSAPICleanupCredentials = False
```

Comment:
this is an odd one - also set in a test and an example, but never used anywhere (removal looks correct)